peekingduck/pipeline/nodes/model/efficientdet.py

```diff
@@ -23,11 +23,42 @@
 
 
 class Node(AbstractNode):
-    """EfficientDet node class that initializes and uses efficientdet model to detect
-    bounding boxes from an image.
+    """EfficientDet node that initalises a EfficientDet model to detect bounding
+    boxes from an image.
 
-    EfficientDet: Scalable and Efficient Object Detection
-    https://arxiv.org/abs/1911.09070
+    The EfficientDet node is capable of detecting objects from 80 categories. The
+    table of categories can be found :term:`here <object detection indices>`.
+    EfficientDet node has five levels of compound coefficient (0 - 5). The
+    default compound coefficient is 0 and can be changed to other values.
+
+    Inputs:
+        |img|
+
+    Outputs:
+        |bboxes|
+
+        |bbox_labels|
+
+        |bbox_scores|
+
+    Configs:
+        model_type (:obj:`int`): **{"0", "1", "2", "3", "4"}, default="0"**
+
+            Defines the compound coefficient for EfficientDet.
```

Comment:
is it possible to give a short explanation of the compound coefficient? or at least what adjusting this config means for the user (slow, but more accurate model?)

Reply:
Noted. Added explanation on the compound coefficient.